src/core/containers.py

```diff
@@ -34,3 +34,25 @@ def __init__(self, flow_name, task_name):
 
     def as_dict(self):
         return {"flow": self.flow, "task": self.task}
+
+
+class Populate(Container):
+    __slots__ = ["path", "validators", "value"]
+
+    def __init__(self, path="", validators="", value="", **kwargs):
```

Comment:
Use `None` instead of empty strings for optional arguments.

Reply:
Done